template.yaml

```diff
@@ -536,6 +560,42 @@ Resources:
               - 's3:List*'
             Resource: '*'
 
+    #---- Buckets ----
+  LoaderBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketName: !Sub "${LoaderBucketName}-${AWS::AccountId}"
+      LifecycleConfiguration:
+        Rules:
+          - Id: DeleteContentAfter3Days
+            Status: Enabled
+            ExpirationInDays: 3
+
+    #---- Role for Neptune load ----
+
+  ReadAndListBucketPolicy:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: [ rds.amazonaws.com ]
+            Action: [ 'sts:AssumeRole' ]
+      Policies:
+        - PolicyName: listGetS3
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:Get*
+                  - s3:List*
+                Resource: !GetAtt LoaderBucket.Arn
```

Comment:
We done it like this in publication-api

`Resource:
    - !Sub !Sub '${LoaderBucket}/*'
`

I think you need the * to tell that it should grant access to all the objects inside the bucket.